test-integration/test-server/src/main/java/org/infinispan/operator/remote/auth/hotrod/HotRodServlet.java

```diff
@@ -29,14 +29,18 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 
          ConfigurationBuilder builder = new ConfigurationBuilder();
          builder.addServer().host(serviceName).port(11222);
-         builder.security().authentication().realm("default").serverName("infinispan").username(username).password(password).enable();
+
+         if(username != null || password != null) {
```

Comment:
Is it really should by OR, not AND?

Reply:
It doesn't really matter in this particular code as the endpoint is triggered from the tests with either both setup or none (so the result will be the same). If we wanted to test what happens if you setup only one of them then OR would be valid as the other would be set to `null`. But that's not in the scope of the Operator tests.